tests/Unit/Http/PolrShortenerTest.php

```diff
@@ -10,7 +10,7 @@
 use GuzzleHttp\Psr7\Response;
 use LaraCrafts\UrlShortener\Http\PolrShortener;
 use LaraCrafts\UrlShortener\Tests\Concerns\HasUrlAssertions;
-use PHPUnit\Framework\TestCase;
+use Orchestra\Testbench\TestCase;
```

Comment:
This is just a quickfix so this test doesn't break on PHPUnit 5, and will be reverted in #29